os/include/net/lwip/sockets.h

```diff
@@ -165,18 +166,14 @@ struct lwip_setgetsockopt_data {
 #error "IOV_MAX larger than supported by lwIP"
 #endif /* IOV_MAX */
 
-#define LWIP_HAVE_UIO	1
-#ifdef LWIP_HAVE_UIO
+#if 0 /* defined in sys/socket.h */
```

Comment:
maybe delete the lines if not required?

Reply:
Thanks for the comment. Yes, these lines are not necessary, but it might be difficult to find the location where 'struct msghdr' is defined, without these lines. Basically, sys/socket.h shows all socket API prototypes based on POSIX, therefore, this structure needs to be implemented there. However, there must be confusion during the LWIP porting.

Reply:
please remove if 0 as we discussed.

Reply:
Modification completed. Thanks for the comment.